pkg/availability/multisigcollector/internal/parts/certcreation/certcreation.go

```diff
@@ -186,6 +186,11 @@ func IncludeCreatingCertificates(
 			logger.Log(logging.LevelWarn, "sender %s is not a member.\n", from)
 			return nil
 		}
+		if txs == nil || sliceutil.Contains(txs, nil) {
```

Comment:
```suggestion
		if len(txs) == 0 || sliceutil.Contains(txs, nil) {
```
(works also for `nil` and catches a non-nil empty list)

Reply:
We talked about moving nil check to computeids.go, so not committing this

Reply:
Sorry I wasn't clear enough about that. What I meant by the `nil` check is checking if any contained transactions are `nil` (second part of this condition), not if the whole list is `nil`. As mentioned in the other comment, I'd suggest doing  only the first part here (`len(txs)` == 0), since it is the availability module that rejects empty lists, and not the mempool. The second part can be removed indeed, since the mempool implements it as well.